server/src/bootstrap/passport/local.strategy.ts

```diff
@@ -0,0 +1,41 @@
+import passport from 'passport'
```

Comment:
does the filename need to be dot-separated? would `localStrategy.ts` make more sense as a filename? `.strategy.ts` doesn't seem to be a reusable extension, unlike something like `.controller.ts`

Reply:
Passport has multiple authentication strategy, and convention is each strategy will reside in a `xxx.strategy.ts` file eg. `facebook.strategy.ts`. Given that we will only use local in the foreseeable future, I think either way is fine.

Reply:
Added tests